src/pool/rpc.c

```diff
@@ -252,6 +252,15 @@ pool_query_bits(daos_pool_info_t *po_info, daos_prop_t *prop)
 		case DAOS_PROP_PO_OWNER_GROUP:
 			bits |= DAOS_PO_QUERY_PROP_OWNER_GROUP;
 			break;
+		case DAOS_PROP_PO_SCRUB_SCHED:
```

Comment:
also wonder if there is a better way to do this...though I guess it's probably not a critical path

Reply:
Not sure what you mean? I just copied what is already being done. Are you thinking of reworking the whole prop query bits approach?